src/intl/resolver.js

```diff
@@ -257,6 +257,9 @@ function* toString(entity) {
 }
 
 export function format(ctx, args, entity) {
+  if (entity === undefined) {
+    return ['', []];
+  }
```

Comment:
Actually, `entityFromContext` should prevent errors here, so I wonder why you saw them at all. Did you try to call call `ctx.format` directly somewhere?